test/any.js

```diff
@@ -147,4 +148,57 @@ describe('parse(req, opts)', function(){
       .expect(415, 'Unsupported Media Type', done);
     })
   })
+
+  describe('with content-encoding', function(){
+    it('should inflate gzip', function(done){
+      var app = koa();
+      var json = JSON.stringify({ foo: 'bar' });
+
+      app.use(function *(){
+        var body = yield parse(this);
+        body.should.eql({ foo: 'bar' });
+        done();
```

Comment:
set parse body to `this.body` and assert body on `request.expect(200, { foo: 'bar' })`